packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -15,8 +15,9 @@ import {Addresslist} from "../addresslist/Addresslist.sol";
 /// @author Aragon Association - 2022.
 /// @notice The on-chain multisig governance plugin in which a proposal passes if X out of Y approvals are met.
 /// @dev This contract inherits from `MajorityVotingBase` and implements the `IMajorityVoting` interface.
-contract Multisig is TimeHelpers, PluginUUPSUpgradeable, Addresslist {
+contract Multisig is PluginUUPSUpgradeable, Addresslist {
     using CountersUpgradeable for CountersUpgradeable.Counter;
+    using SafeCastUpgradeable for uint256;
```

Comment:
Why can't we simply use SafeCast instead? Is this an optimization because this contract is upgradeable?

Reply:
For consistency, if our contract is upgradeable, all of the things we use inside must be upgradeable(with slot gaps).